cmd/server/main.go

```diff
@@ -371,13 +382,15 @@ func pushClientOrDie() *upsClient {
 			ApplicationId: string(upsSecret.Data["applicationId"]),
 		},
 		serviceInstanceId: serviceInstanceId,
-		baseUrl: upsBaseURL,
+		baseUrl:           upsBaseURL,
 	}
 }
 
 func main() {
 	rand.Seed(time.Now().Unix())
 
+	log.Print("Hello this is Dara's custom service")
```

Comment:
I think you can remove that log ;)

Reply:
Good catch! A nice little sanity check to make sure I was actually running my code 😄